omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -44,6 +44,13 @@
       default_type text/plain;
       alias /opt/opscode/version-manifest.txt;
     }
+    <% if node['private_chef']['nginx']['enable_lets_encrypt'] -%>
```

Comment:
Yes for now... just in case anyone decides to use nginx for validation, he/she can create a separate config file that runs on port 80. 
I will amend the instructions and remove the flag, it won't be required for this release.